instructions/1-environment-setup.md

```diff
@@ -1,6 +1,6 @@
 # 1 - Prerequisites
 
-You will need an Red Hat network account to be able to download the JBoss EAP installation zip files.  The account you use must not be managed by SSO in order for the Ansible scripts to be able to authenticate.  You can create a new account by following the instructions in the [Red Hat Customer Portal](https://sso.redhat.com/auth/realms/redhat-external/login-actions/registration?client_id=customer-portal&tab_id=RiPOv96eZ74).  You will need to use these credentials when you run the ansible playbooks later in the workshop.
+You will need an Red Hat network account to be able to download the JBoss EAP installation zip files.  The account you use must not be managed by SSO in order for the Ansible scripts to be able to authenticate.  You can create a new account by following the instructions in the [Red Hat Customer Portal](https://access.redhat.com/?intcmp=701f20000012k6EAAQ).  You will need to use these credentials when you run the ansible playbooks later in the workshop.
```

Comment:
No matter how I tried this link, I get a 500 error page; we could point to the link above, or better we should find a way to point to the registration page